Functions/xASL_num2str.m

```diff
@@ -1,13 +1,12 @@
 function [DataOut] = xASL_num2str(DataIn, f, bConcatenate, strDelimiter)
-%xASL_num2str Wrapper around Matlab builtin 'num2str', bypassing strings/characters
-% & BIDS-compatible
+%xASL_num2str Wrapper around Matlab builtin 'num2str', bypassing strings/characters & BIDS-compatible
 %
 % FORMAT: [DataOut] = xASL_num2str(DataIn[, f, bConcatenate, strDelimiter])
 %
 % INPUT:
 %   DataIn 	     - input data (can be any format) (REQUIRED)
 %   f            - second argument of "num2str". Can be a format, but mostly used as number of characters to print (effectively rounding) (OPTIONAL)
```

Comment:
please add default to f - DEFAULT = ''

Reply:
The DEFAULT for f should be empty, so '[]' (see how this is handled in the rest of the function)

Reply:
ea1ec1d